src/azure-cli/azure/cli/command_modules/storage/operations/azcopy.py

```diff
@@ -14,21 +14,18 @@ def storage_blob_copy(azcopy, source, destination, recursive=None):
     azcopy.copy(source, destination, flags=flags)
 
 
-def storage_blob_upload(cmd, client, source, destination, recursive=None):
-    azcopy = _azcopy_blob_client(cmd, client)
-    storage_blob_copy(azcopy, source, _add_url_sas(destination, azcopy.creds.sas_token), recursive=recursive)
-
-
-def storage_blob_download(cmd, client, source, destination, recursive=None):
-    azcopy = _azcopy_blob_client(cmd, client)
-    storage_blob_copy(azcopy, _add_url_sas(source, azcopy.creds.sas_token), destination, recursive=recursive)
-
-
-def storage_blob_remove(cmd, client, target, recursive=None):
-    azcopy = _azcopy_blob_client(cmd, client)
+def storage_remove(cmd, client, service, target, exclude=None, include=None, recursive=None):
+    if service == 'file':
+        azcopy = _azcopy_file_client(cmd, client)
+    else:
+        azcopy = _azcopy_blob_client(cmd, client)
     flags = []
     if recursive is not None:
         flags.append('--recursive')
```

Comment:
If you just append --recursive in flags, can it work? In azcopy, it should be --recursive=true.

Reply:
'--recursive' equals '--recursive=true'.